tests/e2e_tests/utils.py

```diff
@@ -68,6 +68,58 @@ def sudo_call_set_network_limit(
     return response.is_success
 
 
+def sudo_call_set_weight_limit(
```

Comment:
This needs to be replaced with a BTCLI call within the test itself.

Reply:
will use btcli.

Reply:
I tried the SubnetSudoCommand. but now SubnetSudoCommand just set the parameter for specific netuid. The parameter we need to set is not for any specific netuid. I need a separate command, but didn't find out in the code base.